frontend/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_GOOGLE_REST_API_KEY={type_your_key}
+REACT_APP_NAVER_REST_API_KEY={type_your_key}
+REACT_APP_KAKAO_REST_API_KEY={type_your_key}
```

Comment:
It is better to add empty line at the end of file here and few other places in this PR.